FrontEnd/src/components/basicPage/BasicPage.jsx

```diff
@@ -26,7 +27,7 @@ import TermsAndConditions from '../terms-and-conditions-app/terms_conditions/Ter
 import { useAuth } from '../../hooks';
 import { useUser } from '../../hooks';
 import { Search } from '../SearchPage/Search';
-import classes from './BasicPage.module.css';
+import './customToastStyles.css';
```

Comment:
use module css

Reply:
Overridden toast styles are not applying if use module css.